js/researches/english/passivevoice-english/wordIndicesRegexes.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Matches words from a list in sentence parts and returns them and their indices.
```

Comment:
I'd change this to something like `Matches words in a sentence part. If a match is found, the matched word and index are added to the list of results.`